src/rpc/rawtransaction.cpp

```diff
@@ -916,13 +917,17 @@ static UniValue testmempoolaccept(const JSONRPCRequest& request)
 
     TxValidationState state;
     bool test_accept_res;
+    CAmount fee; // To return transaction fee from AcceptToMemoryPool
```

Comment:
nit: no need for this inline comment. It's clear from the code what this variable is doing.

Reply:
Removed this comment.